src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const semicolon = ';';
+  const colon = ':';
```

Comment:
[CODE STYLE] - Avoid variable namings like `semicolon`, `colon` which are too literal and do not add to the readability or understanding of the code. Instead, use meaningful names that describe the role of the variable in the context of the function.